package.json

```diff
@@ -107,8 +107,9 @@
     "@types/sharp": "0.29.3",
     "@types/string-hash": "1.1.1",
     "@types/trusted-types": "2.0.3",
-    "@typescript-eslint/eslint-plugin": "6.14.0",
-    "@typescript-eslint/parser": "6.14.0",
+    "@types/webpack-sources": "3.2.3",
```

Comment:
`@types/webpack-sources` is added under `packages/next/package.json`, maybe `@types/webpack-sources3` should be, too?

https://github.com/vercel/next.js/blob/3951c7be4fddabfb6d02811c7c1bd008a6b14a44/packages/next/package.json#L194